src/plugin.ts

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import fs = require('fs');
+import path = require('path');
+import sizeTree = require('./size_tree');
+
+export class WebpackBundleSizeAnalyzerPlugin {
+  filepath: string;
+  statsOptions: object;
+  constructor(filepath: string = '', statsOptions: object = {}) {
+    this.filepath = filepath;
+    this.statsOptions = statsOptions || {};
+  }
+  apply(compiler: any) {
+    const that = this;
```

Comment:
Isn't this line redundant given that we're using arrow functions everywhere so `this` is always captured lexically?